crates/noirc_evaluator/src/ssa/optimizations.rs

```diff
@@ -94,7 +96,43 @@ fn evaluate_intrinsic(
                     let i = Instruction::new(op, ObjectType::NotAnObject, Some(block_id));
                     result.push(ctx.add_instruction(i));
                 }
-                return result;
+                return Ok(result);
+            }
+            unreachable!();
```

Comment:
one comment -- can you specify why this is unreachable in the macro : `unreachable!("add text here")`